ingest/rules/join_genotype_metadata.smk

```diff
@@ -0,0 +1,41 @@
+"""
+This part of the workflow pulls genotype data from Genomic Detective and VIPR, cleans the data, and joins it together.
+"""
+
+rule prepare_genotype_metadata:
+    input:
+        metadata = expand("../data/genomicdetective_results{i}.csv", i = [1,2,3])
```

Comment:
This part is still open to discussion, but yes. 😄 I'll push for eventually replacing the genomic detective hardcoded files with a Nextclade dataset [as suggested here](https://github.com/blab/norovirus/issues/2#issuecomment-2406207024).

Creating a Nextclade dataset will require digging into clade-defining mutations or creating a guide tree (spiked in the genome detective results (`ORF1_type` and `ORF2_type`) + `augur traits`). From a [rough visual exploration of the samples](https://github.com/j23414/generated-reports/blob/main/reports/norovirus.pdf), we have GenomeDetective results for approximately `1900` of `51047` norovirus samples with year information.

Reply:
Gotcha, thanks for the background and data exploration! 

In a brief search, I found the [CDC typing tool](https://www.rivm.nl/mpf/typingtool/norovirus/) and [RIVM typing tool](https://www.rivm.nl/mpf/typingtool/norovirus/), but both are web-only tools. Totally agree with push for Nextclade dataset to make automation easier, especially since there's a [good list of reference sequences](https://calicivirustypingtool.cdc.gov/becerance.cgi) by the CDC.

Reply:
Wrote up an isssue to track this work: https://github.com/nextstrain/norovirus/issues/6